pkg/reporting/preflight.go

```diff
@@ -68,30 +69,67 @@ func SendPreflightInfo(appID string, sequence int, isSkipPreflights bool, isUpda
 
 	clusterID := downstreams[0].ClusterID
 
-	if isSkipPreflights || isUpdate {
-		// at this point current version status does not exist so it's
-		// neccessary to create thread to get it after version is deployed
+	go func() {
+		appStatus := appstatustypes.StateMissing
+		for start := time.Now(); time.Since(start) < 20*time.Minute; {
+			s, err := store.GetStore().GetAppStatus(appID)
+			if err != nil {
+				logger.Debugf("failed to get app status: %v", err.Error())
```

Comment:
I know this isnt new code but shouldnt this be at warn level?

Reply:
What should be warn level debug?
I'm seeing only Debug, Error, Debugf and Info

Reply:
I'll let @salahalsaleh chime in here. I just assume these are real errors and they are being masked cause they are debug level. What is the reason for ignoring them? This is old code so not a blocker for merging.

Reply:
i remember the product decision was that we shouldn't log "reporting/metrics" errors at error level